setup.py

```diff
@@ -19,19 +19,95 @@
+
+    The authors are written in AUTHORS as bullet points under the
+    "Chronological list of authors" title.
+    """
+    authors = []
+    with codecs.open(abspath('AUTHORS'), encoding='utf-8') as infile:
+        # An author is a bullet point under the title "Chronological list of
+        # authors". We first want move the cursor down to the title of
+        # interest.
+        for line_no, line in enumerate(infile, start=1):
+            if line.rstrip() == "Chronological list of authors":
+                break
+        else:
+            # If we did not break, it means we did not find the authors.
+            raise IOError('EOF before the list of authors')
+        # Skip the next line as it is the title underlining
+        line = next(infile)
+        line_no += 1
+        if line[:4] != '----':
+            raise IOError('Unexpected content on line {0}, '
+                          'should be a string of "-".'.format(line_no))
+        # Add each bullet point as an author
+        for line in infile:
+            if line.strip()[:2] == '- ':
+                # This is a bullet point, so it should be an author name.
+                name = line.strip()[2:].strip()
+                # Remove any @-handles in "A.U. Thor (@donnergott)"
+                name = name.split("(")[0].strip()
+                authors.append(name)
+
+    # So far, the list of authors is sorted chronologically. We want it
+    # sorted alphabetically of the last name.
+    authors.sort(key=lambda name: name.split()[-1])
+
+    # Write the authors.py file.
+    out_path = abspath('MDAnalysisData/authors.py')
+    with codecs.open(out_path, 'w', encoding='utf-8') as outfile:
+        # Write the header
+        header = '''\
+#-*- coding:utf-8 -*-
+
+# This file is generated from the AUTHORS file during the installation process.
+# Do not edit it as your changes will be overwritten.
+'''
+        print(header, file=outfile)
+
+        # Write the list of authors as a python list
+        template = u'__authors__ = [\n{}\n]'
+        author_string = u',\n'.join(u'    u"{}"'.format(name)
+                                    for name in authors)
+        print(template.format(author_string), file=outfile)
+
+
+
 if __name__ == '__main__':
     with codecs.open("README.md", encoding="utf-8") as summary:
         LONG_DESCRIPTION = summary.read()
 
+    try:
+        dynamic_author_list()
+    except (OSError, IOError):
+        warnings.warn('Cannot write the list of authors.')
+
     CLASSIFIERS = [
         'Development Status :: 4 - Beta',
         'Environment :: Console',
         'Intended Audience :: Science/Research',
         'License :: OSI Approved :: BSD License',
         'Operating System :: POSIX',
         'Operating System :: MacOS :: MacOS X',
+        'Operating System :: Microsoft :: Windows ',
         'Programming Language :: Python',
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
```

Comment:
Since MDA has dropped 2.7 & 3.5 should we be dropping it here too? (in any case testing for those versions would be nearly impossible now)